cool_seq_tool/sources/uta_database.py

```diff
@@ -866,37 +866,41 @@ async def get_gene_from_ac(
     async def get_transcripts_from_gene(
         self,
         gene: str,
-        start_pos: int,
-        end_pos: int,
+        start_pos: Optional[int] = None,
+        end_pos: Optional[int] = None,
         use_tx_pos: bool = True,
         alt_ac: Optional[str] = None,
     ) -> pd.core.frame.DataFrame:
         """Get transcripts associated to a gene.
 
         :param str gene: Gene symbol
```

Comment:
Since you removed the other types in the docstrings, go ahead and do this one too

```suggestion
        :param gene: HGNC gene symbol
```

Reply:
added these back to be consistent w the rest of the file and functions. this was unintentional (leftover from the commit that accidentally got in here maybe?)

Reply:
We actually have been removing types from docstrings (slowly) since they're in the method signature. You can remove.